tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_022_pos.ksh

```diff
@@ -74,6 +75,11 @@ set -A RW_FS_PROP "quota=512M" \
 typeset -i i=0
 typeset opts=""
 
+typeset disk="$($ECHO $DISKS | $NAWK '{print $1}')"
+if is_mpath_device $disk; then
+	delete_partitions
+fi
```

Comment:
Rather than adding this hunk to each test perhaps it should be moved to the `is_linux` block of `zpool_create.cfg`.